src/main.cpp

```diff
@@ -771,13 +833,40 @@ struct capture_region
 {
     int32_t x, y;
     int32_t width, height;
+    int transform = WL_OUTPUT_TRANSFORM_NORMAL;
 
     capture_region()
         : capture_region(0, 0, 0, 0) {}
 
     capture_region(int32_t _x, int32_t _y, int32_t _width, int32_t _height)
         : x(_x), y(_y), width(_width), height(_height) { }
 
+    void set_transform(const wf_recorder_output& wo) {
+        transform = wo.transform;
+        switch (transform) {
+            case WL_OUTPUT_TRANSFORM_NORMAL:
+            case WL_OUTPUT_TRANSFORM_180:
+            case WL_OUTPUT_TRANSFORM_FLIPPED:
+            case WL_OUTPUT_TRANSFORM_FLIPPED_180:
+            case WL_OUTPUT_TRANSFORM_90:
+                fprintf(stderr, "90 degree rotation detected, altering geometry to compensate.\n");
```

Comment:
This doesn't seem right. It says 90 but all the cases above it fall through to this message.